server/getCredentials.ts

```diff
@@ -1,17 +1,26 @@
 "use server";
 
 import { Credentials } from "@/app/types";
- const validLoginDetails: Credentials[] = [
-  {
-    username: "FakeUser1",
-    password: "Password1",
-  },
-  {
-    username: "FakeUser2",
-    password: "Password2",
-  },
-];
 
-export async function getCredentials(): Promise<Credentials[]> {
-  return validLoginDetails;
+export async function credentialsAreValid(
+  inputUsername: string,
+  inputPassword: string
+) {
+  const validLoginDetails: Credentials[] = [
+    {
+      username: "FakeUser1",
+      password: "Password1",
+    },
+    {
+      username: "FakeUser2",
+      password: "Password2",
+    },
+  ];
```

Comment:
any reason for this const to live inside the function? doing so means it's recreated every time the function runs but doesn't look like it changes

Reply:
okay moved it out